OONIProbeUITests/OONIProbeUITests.swift

```diff
@@ -115,6 +115,7 @@ class OONIProbeUITests: XCTestCase {
 
         dashboardButton.tap()
         tablesQuery.cells.element(boundBy: 0).tap()
+        //Configure button doesn't exist anymore
```

Comment:
here the app should go inside the card and tap to custom url, as the configure button doesn't exist anymore